model.js

```diff
@@ -49,5 +50,15 @@ return db.query(fetchArtIdStr, [article_id])
 });
 };
 
+const fetchComByArt = (article_id) => {
```

Comment:
I would be more explicit with the name, rather than abbreviating com, use comments, it will make it easier to understand just looking from the controller.